Update ug (#205)
AY2021S1-CS2103T-T10-4/tp: docs/UserGuide.md

#### Listing all classes : `list-class`

> If you would like to see a list of all classes in the application, make use of this command.

Reviewer: ```suggestion
If you would like to see a list of all classes in your Tutor's Pet, make use of this command.
```

You can use this command to edit the attendance of a student.

Format: `edit-attendance c\CLASS_INDEX l\LESSON_INDEX s\STUDENT_INDEX w\WEEK p\PARTICIPATION_SCORE`
* Edits the attendance record at the specified `CLASS_INDEX` + `LESSON_INDEX` + `STUDENT_INDEX` + `WEEK`.

For example:
* Alex is a student in your CS2103T class. After your lesson in week 1, you wish to give him a participation score of 45.

>   However, you made a typo error and gave him 35 instead. 

Reviewer: I think typo already suggests that its an error? 

```suggestion
  However, you made a typo and gave him 35 instead. 
```
Alternatively maybe we can drop the typo word?

```suggestion
  However, you accidentally gave him a score of 35 instead.
```
